src/python/grpcio_health_checking/grpc_health/v1/health.py

```diff
@@ -85,19 +101,26 @@ def Check(self, request, context):
             else:
                 return _health_pb2.HealthCheckResponse(status=status)
 
-    def Watch(self, request, context):
+    # pylint: disable=arguments-differ
+    def Watch(self, request, context, on_next=None):
```

Comment:
I understand this is an internal API, but do you think we need a more readable name for the `on_next`?

Reply:
More readable is always better. I updated to `on_next_callback`, but not sure that's that much of an improvement. What would you suggest?

Reply:
`on_next_callback` seems not directly bound with request or response, how about `on_response`, `send_response` or even "response_observer"? I hope another engineer who doesn't join our offline discussion could understand what it means quickly.

Reply:
Thanks for the suggestions. I went with `send_response_callback` (`send_response` would be fine as well, but I wanted the name to be consist with the name of the `set` that they are added to, and `send_responses` seems unclear compared to `send_response_callbacks`)